docs/source/models.rst

```diff
@@ -87,6 +90,61 @@ signature
 input_example
   reference to an artifact with :ref:`input example <input-example>`.
 
+databricks_runtime
+    Databricks runtime version and type, if the model was trained in a Databricks notebook or job.
```

Comment:
@andreakress do we need to mention anything about which version of mlflow client this was added in, or reflecting the current state is fine?

Reply:
So, for the OS docs, this will appear with the new release, right (1.18.0?). If I look at earlier doc versions, it will not appear? If so, no need to add the version it was added in.

Reply:
That's right - i forgot you can look at earlier versions of docs. That makes sense!